docs/support/troubleshooting.md

```diff
@@ -152,16 +152,15 @@ When running the Planning Simulator, the most common reason for the map not bein
 
 Another possible reason is that map loading is taking a long time due to poor DDS performance. To address this issue, first enable localhost-only communication to reduce network traffic, and then [tune DDS settings](https://docs.ros.org/en/rolling/How-To-Guides/DDS-tuning.html) if the problem continues to occur.
 
-1. [Enable localhost-only communication](../installation/additional-settings-for-developers/index.md#enabling-localhost-only-communication)
-2. Tune DDS settings
-
-Add the following lines to `/etc/sysctl.conf`
+Simply put, add the following settings to `.bashrc` and reboot the terminal. In many cases this is not a problem.
 
 ```bash
-net.ipv4.ipfrag_time=3  // generic DDS setting
-net.ipv4.ipfrag_high_thresh=134217728 // generic DDS setting
-net.core.rmem_max=2147483647 // only add if CycloneDDS is configured
-net.core.rmem_default=8388608 // only add if CycloneDDS is confgured
+export ROS_LOCALHOST_ONLY=1
+if [ ! -e /tmp/cycloneDDS_configured ]; then
+  sudo sysctl -w net.core.rmem_max=2147483647
+  sudo ip link set lo multicast on
+  touch /tmp/cycloneDDS_configured
+fi
```

Comment:
Memo: Added by https://github.com/autowarefoundation/autoware-documentation/pull/107.

For this part, I want a review from @takam5f2.

Reply:
okay :+1: 

Memo: AWSIM write this code in the tutorial.
https://tier4.github.io/AWSIM/GettingStarted/QuickStartDemo/

Reply:
@takam5f2 is it ok?